### PR TITLE
Create matcher to troubleshoot leaks.

### PR DESCRIPTION
**Pull request by polina-c**
Create matcher to troubleshoot leaks.

https://user-images.githubusercontent.com/12115586/208321882-ecb96152-3aa7-4671-800e-f2eb8c18149e.mov

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +243,37 @@ See `DevTools > Memory > Leaks` guidance on how to interact with leak tracker.
 
 TODO: add link to DevTools documentation.
 
+## Troubleshoot leaks
+
+### Collect callstack
+
+The stacktrace at the moments of tacking start and helps to find owner, that did
+not invoke dispose, i.e. not-disposed leaks.
```
[the remark this answers is not in the document]
Comment (polina-c) [reply README.md:251]: Indeed. Thank you!

File: README.md [hunk 2]
```diff
@@ -243,6 +243,37 @@ See `DevTools > Memory > Leaks` guidance on how to interact with leak tracker.
...
+There are options to enable stacktrace collection for troubleshooting:
+
+1. By passing `stackTraceCollectionConfig`
+to `withLeakTracking` or `enableLeakTracking`.
+
+<video src="md_images/collect_callstack.mov" controls="controls" playsinline="playsinline"></video>
+
+2. Using interactive UI in DevTools > Memory > Leaks.
+
+TODO: add details
```
[the remark this answers is not in the document]
Comment (polina-c) [reply README.md:268]: Yes, we need to explain how to use DevTools.

[the remark this answers is not in the document]
Comment (polina-c) [reply README.md:267]: Added explanation.

File: README.md [hunk 3]
```diff
@@ -243,6 +243,37 @@ See `DevTools > Memory > Leaks` guidance on how to interact with leak tracker.
...
 TODO: add link to DevTools documentation.
 
+## Troubleshoot leaks
+
+### Collect callstack
+
+The stacktrace at the moments of tacking start and helps to find owner, that did
+not invoke dispose, i.e. not-disposed leaks.
+
+And, the stacktrace at the moment of disposal helps to find non-nulled long living
```
[the remark this answers is not in the document]
Comment (polina-c) [reply README.md]: not applicable any more

File: README.md [hunk 4]
```diff
@@ -247,16 +247,15 @@ TODO: add link to DevTools documentation.
 
 ### Collect callstack
 
-The stacktrace at the moments of tacking start and helps to find owner, that did
-not invoke dispose, i.e. not-disposed leaks.
-
-And, the stacktrace at the moment of disposal helps to find non-nulled long living
-references, i.e. not-GCed leaks.
+Stacktrace for the object's lifecycle events may help to catch out
+the leak's root cause. The lyfecycle event will be creation
```
Comment (CoderDake) [inline README.md:251]:
```suggestion
the leak's root cause. The lifecycle event will be creation
```

**Review by CoderDake — APPROVED** [review]

**Pull request merged**